### PR TITLE
Update default_app to ES6 conventions

### PR DESCRIPTION
**Pull request by stevekinney**
Updates the default application to the same conventions as found in the latest revisions of the documentation.

### CODE REVIEW

File: default_app/default_app.js [hunk 1]
```diff
@@ -1,16 +1,15 @@
 const electron = require('electron')
-const app = electron.app
-const BrowserWindow = electron.BrowserWindow
+const {app, BrowserWindow} = electron
```
Comment (kevinsawicki) [inline default_app/default_app.js]: Similar comment here, perhaps just make this `const {app, BrowserWindow} = require('electron')` and remove line 1?

File: default_app/index.html [hunk 2]
```diff
@@ -114,8 +114,7 @@
 <body>
   <script>
     const electron = require('electron');
-    const remote = electron.remote;
-    const shell = electron.shell;
+    const {remote, shell} = electron;
```
Comment (stevekinney) [inline default_app/index.html]: It totally could and _should_ be. I had text editor saving issue. Thanks for catching it!

File: default_app/main.js [hunk 3]
```diff
@@ -93,25 +90,25 @@ app.once('ready', function () {
...
           }
         },
         {
           label: 'Toggle Developer Tools',
-          accelerator: (function () {
+          accelerator: (() => {
             return (process.platform === 'darwin') ? 'Alt+Command+I' : 'Ctrl+Shift+I'
           })(),
-          click: function (item, focusedWindow) {
+          click(item, focusedWindow) {
```
Comment (kevinsawicki) [inline default_app/main.js]: This is a cool change 👍 Hadn't thought about using this new function syntax in object literals before, usually only use it in classes.

Comment (stevekinney) [inline default_app/main.js]: Thanks—I really like it, too. That said, the linting process in CI doesn't like it—even though it's valid code. Do you think I should change the code to appease the linter?

**Comment by zcbenz** [comment]
Thanks!

**Pull request merged**